test/integration/controller/jobset_controller_test.go

```diff
@@ -122,9 +122,8 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 	}
 
 	type testCase struct {
-		makeJobSet               func(*corev1.Namespace) *testing.JobSetWrapper
-		jobSetCreationShouldFail bool
```

Comment:
is this removed because it isn't used anywhere anymore?

Reply:
Yea

Reply:
https://github.com/kubernetes-sigs/jobset/pull/254#discussion_r1294042022